src/install-choose/install-choose-dialog.ts

```diff
@@ -169,53 +165,6 @@ class ESPHomeInstallChooseDialog extends LitElement {
                 }}
               ></mwc-button>
             `;
-    } else if (this._state === "pick_download_type") {
-      heading = "What version do you want to download?";
-      content = html`
-        <mwc-list-item
-          twoline
-          hasMeta
-          dialogAction="close"
-          @click=${this._handleWebDownload}
-        >
-          <span>Modern format</span>
-          <span slot="secondary">
-            For use with ESPHome Web and other tools.
-          </span>
-          ${metaChevronRight}
-        </mwc-list-item>
-
-        <mwc-list-item
-          twoline
-          hasMeta
-          dialogAction="close"
-          @click=${this._handleManualDownload}
-        >
-          <span>Legacy format</span>
-          <span slot="secondary">For use with ESPHome Flasher.</span>
-          ${metaChevronRight}
-        </mwc-list-item>
-
-        ${this._platformSupportsWebSerial
-          ? html`
-              <a
-                href="https://web.esphome.io"
-                target="_blank"
-                rel="noopener noreferrer"
-                class="bottom-left"
-                >Open ESPHome Web</a
-              >
-            `
-          : ""}
```

Comment:
Please make sure this is not lost for ESP based devices

Reply:
Done.